packages/create-ripple-app/generators/updates/1.32.0/saofile.js

```diff
@@ -0,0 +1,90 @@
+const jscodeshift = require('jscodeshift')
+
+module.exports = {
+  ...common,
+  templateDir,
+  actions () {
+    const actions = [
+      {
+        type: 'modify',
+        files: 'package.json',
+        handler (data) {
+          data.dependencies['nuxt'] = `2.14.12`
+          data.devDependencies['@babel/eslint-parser'] = `^7.12.1`
+          data.devDependencies['@babel/eslint-plugin'] = `^7.12.1`
+          data.devDependencies['cross-env'] = `^7.0.3`
+          data.devDependencies['eslint'] = `^7.19.0`
+          data.devDependencies['eslint-config-standard'] = `^16.0.2`
+          data.devDependencies['eslint-friendly-formatter'] = `^4.0.1`
+          data.devDependencies['eslint-loader'] = `^4.0.2`
+          data.devDependencies['eslint-plugin-jest'] = `^24.1.3`
+          data.devDependencies['eslint-plugin-node'] = `^11.1.0`
+          data.devDependencies['eslint-plugin-promise'] = `^4.2.1`
+          data.devDependencies['eslint-plugin-standard'] = `^5.0.0`
+          data.devDependencies['eslint-plugin-vue'] = `^7.5.0`
+          // Delete babel-eslint as this is now replaced with @babel/eslint-parser
+          delete data.devDependencies['babel-eslint']
+          return data
+        }
+      },
+      {
+        type: 'modify',
+        files: '.eslintrc.js',
+        handler (data) {
+          const j = jscodeshift
+
+          // Update eslint parser from babel-eslint to @babel/eslint-parser
+          let result = j(data)
+            .find(j.Literal)
+            .filter(path => path.node.value === 'babel-eslint')
+            .forEach(path => {
+              j(path).replaceWith(j.literal('@babel/eslint-parser'))
+            }).toSource({ quote: 'single' })
+
+          // Add new rules
+          const rulesData = j(result).find(j.Property, { key: { name: 'rules' } })
+          const n = rulesData.length
+
+          if (n) {
+            const newRules = `// TODO enable the rules to achieve lint standard consistency towards projects
+'array-bracket-spacing': 'off',
+'array-callback-return': 'off',
+'dot-notation': 'off',
+'jest/expect-expect': 'off',
+'jest/no-standalone-expect': 'off',
+'jest/no-try-expect': 'off',
+'jest/no-conditional-expect': 'off',
+'lines-between-class-members': 'off',
+'multiline-ternary': 'off',
+'no-case-declarations': 'off',
+'no-prototype-builtins': 'off',
+'node/no-deprecated-api': 'off',
+'prefer-const': 'off',
+'prefer-regex-literals': 'off',
+'quotes': ['off', 'single', { 'allowTemplateLiterals': true }],
+'quote-props': 'off',
+'vue/no-mutating-props': 'off',
+'vue/no-unused-components': 'off',
+'vue/no-use-v-if-with-v-for': 'off',
+'vue/return-in-computed-property': 'off',
+'no-trailing-spaces': 'off',
+'object-curly-spacing': 'off',
+'no-var': 'off'`
+            result = rulesData.find(j.ObjectExpression).forEach(p => {
+              p.get('properties').push(newRules)
+            })
+          }
+
+          return result ? result.toSource({ quote: 'single' }) : false
```

Comment:
@tim-yao I have tested this in vic-gov-au and aboriginal-vic, they both works fine. Also tested in coronavirus-vic but it is throwing an error as the project doesn't have babel.config.js - will look into this if there is a way to not require babel config in a website.

Reply:
Hi @rubyllamadora Nuxt doesn't require a babelrc file by default: https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-build#babel

And Ripple use babel.config.js is because this is a mono repo. See explain here https://babeljs.io/docs/en/configuration#whats-your-use-case

So I guess something is wrong? However, when I run jest to do unit testing, it will require babel config so I added one for aboriginal-vic which is just for test environment.

Reply:
@rubyllamadora Looks like `babel-eslint-parser` has `requireConfigFile: true,` as default. However Nuxt is using its build in babel config. Some projects like vic-gov-au has a babel config file is just for testing, not for Nuxt. I think we need to set `requireConfigFile: false,` so we can prevent this error.

There are two Nuxt module worth to watch in the future:
https://github.com/nuxt/eslint-config
https://github.com/nuxt/eslint-plugin-nuxt

Reply:
Thanks @tim-yao, good to know regarding babel config setup in vic-gov. I'll work on disabling `requireConfigFile`.